pkg/network/admitter/slirp_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
+	"kubevirt.io/kubevirt/pkg/libvmi"
```

Comment:
nit: Please place this in the same group as `"kubevirt.io/kubevirt/pkg/network/admitter"`.